### PR TITLE
create contact/profile component

### PR DESCRIPTION
**Pull request by Ponomaleks**

### CODE REVIEW

File: src/app/main/components/contact/contact.component.html [hunk 1]
```diff
@@ -1 +1,79 @@
...
+    class="action-button profile__edit-button"
+    [class]="[buttonThemes.Secondary, buttonThemes.Small]"
+    [additionalStyles]="{ width: '100%' }"
+    (click)="onOpenEdit()"
+    >Edit Profile
+  </epm-button>
+
+  <epm-button *ngIf="!personalId" [class]="buttonThemes.Primary" (click)="onOpenChat(user)" class="action-button"
+    >Message
+  </epm-button>
```
Comment (GennadiyShpak) [inline src/app/main/components/contact/contact.component.html:40]:
```suggestion
  <epm-button
    *ngIf="personalId"
    class="action-button profile__edit-button"
    [class]="[buttonThemes.Secondary, buttonThemes.Small]"
    [additionalStyles]="{ width: '100%' }"
    (click)="onOpenEdit()"
    >
    Edit Profile
  </epm-button>

  <epm-button 
  *ngIf="!personalId"
   class="action-button"
   [class]="buttonThemes.Primary"
    (click)="onOpenChat(user)"
    >
    Message
  </epm-button>
```
Minor but

Comment (Ponomaleks) [reply src/app/main/components/contact/contact.component.html]: changed

File: src/app/main/components/chat/chat.component.ts [hunk 2]
```diff
@@ -75,19 +74,17 @@ export class ChatComponent implements OnInit, AfterViewInit {
...
     return message.id;
   }
 
   private setTitles(): void {
-    if (this.chat) {
-      this.title.setTitle(`U-PAMERS | ${this.chat.name} ${this.chat.surname} chat`);
-      this.headerService.addTitle(`${this.chat.name} ${this.chat.surname}`);
-    }
+    this.title.setTitle(`U-PAMERS | ${this.interlocutor.name} ${this.interlocutor.surname} chat`);
+    this.headerService.addTitle(`${this.interlocutor.name} ${this.interlocutor.surname}`);
```
Comment (GennadiyShpak) [inline src/app/main/components/chat/chat.component.ts:87]:
```suggestion
    const fullName = `${this.interlocutor.name} ${this.interlocutor.surname}`
    this.title.setTitle(`U-PAMERS | ${fullName} chat`);
    this.headerService.addTitle(fullName);
```

File: src/app/main/components/contact-interests/contact-interests.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    if (!this.interests.length) return;
+
+    this.chipsArray = this.interests.map(interest => {
+      return {
+        chipsData: INTEREST_CHIPS_LIST[INTEREST_CHIPS_NAME[interest as keyof typeof INTEREST_CHIPS_NAME]],
+        isEditable: false,
+        isActive: true
+      };
+    });
+  }
```
Comment (GennadiyShpak) [inline src/app/main/components/contact-interests/contact-interests.component.ts:35]:
```suggestion
    this.chipsArray = this.interests.map(interest => ({
        chipsData: INTEREST_CHIPS_LIST[INTEREST_CHIPS_NAME[interest as keyof typeof INTEREST_CHIPS_NAME]],
        isEditable: false,
        isActive: true
    });
  }
```

File: src/app/main/pages/contact-list/contact-list.component.ts [hunk 4]
```diff
@@ -31,8 +31,8 @@ export class ContactListComponent implements OnInit, OnDestroy {
     this.mainApiService.clearUserList();
   }
 
-  onOpenUser(user: ExpandedUserDetailed) {
-    this.router.navigateByUrl(`${APP_ROUTER_NAME.Main}/${APP_ROUTER_NAME.Contact}/${user.userId}`, { state: user });
+  onOpenUser(userId: string) {
```
Comment (GennadiyShpak) [inline src/app/main/pages/contact-list/contact-list.component.ts:34]: where returned type?

Comment (Ponomaleks) [reply src/app/main/pages/contact-list/contact-list.component.ts]: fixed

File: src/app/main/components/contact-interests/contact-interests.component.ts (hunk 3 above)
Comment (Ponomaleks) [reply src/app/main/components/contact-interests/contact-interests.component.ts]: Thanks. Fixed as advised

File: src/app/main/components/chat/chat.component.ts (hunk 2 above)
Comment (Ponomaleks) [reply src/app/main/components/chat/chat.component.ts]: fixed

**Review by GennadiyShpak — APPROVED** [review]

**Pull request merged**